scipy-ar-2013/videos/scipyconar-2013-python-cientifico-episodio-i-damian-avila-dia-1-de-3.json

```diff
@@ -0,0 +1,15 @@
+{
+  "description": "",
```

Comment:
same as for Juan Pedro Fisanotti's 3 days, we can probably put text from
http://scipyla.org/conf/2013/activities/index.html > Python Científico: Episodio I - Damián Avila